developersociety/commonslibrary
# Replace and style The Campaigns Library header text

# Sources
https://trello.com/c/guqW3oIB/23-can-we-please-have-the-words-the-campaigns-library-written-across-the-top-of-page

# Description
This PR replaces the original 'Hello there' header text with the text 'The Campaigns Library' and adds styling similar to the design inspiration provided. 

# How to test
Go to http://127.0.0.1:8000/ and check that the next 'The Campaigns Library' header text looks right in terms of text and styling. I have made the border and alignment as similar to the inspiration screenshot as possible, but as there is no subheader to break the border and the colour palette is different I've gone with some of my own initiative of what looks good here. Please let me know if you have any suggestions!

For comprehensive guidelines on code review at DEV, see here: https://www.notion.so/developersociety/Code-Review-f804361f9b6b4859bdcd69d7704168e5.

# Pre-review checklist
